css/image.css

```diff
@@ -0,0 +1,53 @@
+#image{
+    height:115rem;
+    width: 100%;
+    padding: 2rem;
+    background-color: rgb(255, 255, 255);
+    font-family: 'Hind',sans-serif;
+}
+.image-sample-cont{
+    height: 25rem;
+    width: 90%;
+    margin-top: 2rem;
+    border-radius: 10px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    border: 3px solid #524ea1;
+}
+.img-res{
+    height: 90%;
```

Comment:
The image is not proper as it is getting displayed for more than its width. Choose a different image maybe?